criu/util.c

```diff
@@ -1312,6 +1346,13 @@ int setup_tcp_server(char *type)
 		goto out;
 	}
 
+#ifdef SO_ZEROCOPY
```

Comment:
An existence of SO_ZEROCOPY doesn't guarantee that zerocopy is supported by kernel and vice-versa.